customcontext/customcontext.pro

```diff
@@ -3,10 +3,10 @@ TARGET=customcontext
 
 CONFIG += plugin
 
-contains(QT_VERSION, ^5\\.[2-9]\\..*) {
+equals(QT_MAJOR_VERSION, 5):greaterThan(QT_MINOR_VERSION, 1) {
```

Comment:
The whole customcontext_qt520 looks like something that can be removed now and the parts of the file with rules for < 5.2 qt.

Reply:
Fixed with https://github.com/sailfishos/qtscenegraph-adaptation/pull/1/commits/a70e9a9e883cd0e6e989e18462b33d356ad395ce